### PR TITLE
Imagecomponent

### LINKED ISSUE #15: Create image component
Create the image component that:
- takes all available space(width & height = 100%)
- resizes the image to fill the full size

### PR DESCRIPTION
**Pull request by loredanaspivac**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (vbetivu-ellation) [inline .vscode/launch.json:1]: Let's remove this file as it is ide-related.

File: image_component.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+
+<link rel="stylesheet" href="stylesheet.css"></link>
+
+const img = document.createElement("img");
+img.src = "https://beta.crunchyroll.com/imgsrv/display/thumbnail/1200x675/catalog/crunchyroll/6e5065403981315750fa9bd117365cb7.jpg";
+document.body.appendChild(img);
+
```
Comment (vbetivu-ellation) [inline image_component.js:8]: Let's follow the folder structure: `src/components/common/Image/Image.js`

File: image_component.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
```
Comment (vbetivu-ellation) [inline image_component.js:1]: Here is a small reference to how a react component looks:https://reactjs.org/docs/components-and-props.html.
The current code is not a react component

File: imagecomponent.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+interface OuterProps {
```
Comment (vbetivu-ellation) [inline imagecomponent.js:1]: Let's remove this file as it does not work

File: src/components/like_svg.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
```
Comment (vbetivu-ellation) [inline src/components/like_svg.js:1]: This component is a part of different PR, let's remove it here

File: stylesheet.css [hunk 6]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (vbetivu-ellation) [inline stylesheet.css:1]: The CSS should be within the `Image.module.css` file `src/components/common/Image/Image.module.css`.
Placed next to the `Image.js` file

File: stylesheet.css [hunk 7]
```diff
@@ -0,0 +1,11 @@
+
+img 
+{ object-fit: fill;
+
+    width = "100px";
+    height = "100px";
+    border = "0px";
```
Comment (vbetivu-ellation) [inline stylesheet.css:7]: CSS has a different syntax. Check the properties syntax here https://www.w3schools.com/css/

**Review by vbetivu-ellation — COMMENTED** [review]
Please reach out in slack if you need any help.

File: image.js [hunk 8]
```diff
@@ -0,0 +1,51 @@
+
+
+import React from 'react';
+import { View, Image, StyleSheet } from 'react-native';
```
Comment (darsenii-crunchyroll) [inline image.js:4]: We use React js not react-native

**Comment by vbetivu-ellation** [comment]
Closing this PR as Image component was already merged as a part of https://github.com/vbetivu-ellation/crunchygram/pull/60

**Pull request closed without merging**